src/timeline.js

```diff
@@ -852,6 +852,14 @@ export default class Timeline extends React.Component {
         return width - groupOffset;
       };
     }
+
+    function calculateHeight(height) {
+      if (height === 0) return height;
+      // substract timebar height from total height
+      const timebarHeight = document.querySelector(`.rct9k-id-${componentId} .rct9k-timebar`).getBoundingClientRect().height;
+      return height - timebarHeight;
+    }
```

Comment:
I doubt its possible `timebarHeight` is > `height`, but probably better to be safe:
```suggestion
    function calculateHeight(height) {
      // substract timebar height from total height
      const timebarHeight = document.querySelector(`.rct9k-id-${componentId} .rct9k-timebar`).getBoundingClientRect().height;
      return Math.max(height - timebarHeight, 0);
    }
```

Reply:
It's a good point. I will commit this suggestion.